integration_tests/test_python_plugin.py

```diff
@@ -83,6 +83,24 @@ def test_build_rewrites_shebangs(self):
         self.assertEqual('#!/usr/bin/env python2', python2_shebang)
         self.assertEqual('#!/usr/bin/env python3', python3_shebang)
 
+    def test_build_does_not_keep_pyc_or_pth_files_in_install(self):
+        # .pyc and .pyc files collide between parts.
+        # There is no way to tell pip or setup.py to disable generation of
+        # .pyc
+        # The .pth files are only relevant if found inside the pre compiled
+        # site-packges directory so we don't want those either.
+        project_dir = 'pip-requirements-file'
+        self.run_snapcraft('stage', project_dir)
+
+        pyc_files = []
+        pth_files = []
+        for _, _, files in os.walk(os.path.join(project_dir, 'stage')):
+            pyc_files.extend([f for f in files if f.endswith('pyc')])
+            pth_files.extend([f for f in files if f.endswith('pth')])
+
+        self.assertEqual([], pyc_files)
+        self.assertEqual([], pth_files)
```

Comment:
If this fails, it might be much easier to digest an error based on the length of the lists rather than directly comparing their contents (it'll print the lists, right?).